com.unity.render-pipelines.universal/Runtime/ScriptableRenderer.cs

```diff
@@ -773,6 +797,18 @@ void SetRenderPassAttachments(CommandBuffer cmd, ScriptableRenderPass renderPass
                 else
                     finalClearFlag |= (renderPass.clearFlag & ClearFlag.Depth);
 
+                if(!m_FirstCameraRenderPassExecuted)
```

Comment:
This also seems very hacky code from hackweek. We should encapsulate this in get camera clear color and flags and let the pipeline handle that.

Reply:
Agreed - that is pretty unpleasant.

Reply:
This has now gone - there were issues relating to it anyway but it turned out to be unnecessary once I'd refactored the code.